src/FixedDataTable.js

```diff
@@ -346,7 +351,7 @@ var FixedDataTable = React.createClass({
   },
 
   _shouldHandleWheelX(/*number*/ delta) /*boolean*/ {
-    if (this.props.overflowX === 'hidden') {
+    if (this.props.overflowX === 'hidden' || this.props.disableWheelHandler) {
```

Comment:
You also need to handle ReactWheelHandler being initalized here:
https://github.com/schrodinger/fixed-data-table-2/blob/master/src/FixedDataTable.js#L334